Packs/Microsoft365Defender/ReleaseNotes/3_0_0.md

```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### New: O365 Defender SafeLinks - Single User
```

Comment:
shouldn't it be in a new pack like the rest of the `Single User` integrations?

Reply:
discussed in person.